src/commcare_cloud/ansible/roles/kafka/tasks/main.yml

```diff
@@ -55,11 +55,12 @@
 
 - name: Copies kafka log4j.properties
   copy:
-    src: "log4j_{{ kafka_version }}.properties"
+    src: "log4j_2.X.X.properties"
     dest: /opt/kafka/config/log4j.properties
     owner: root
     group: root
     mode: 644
+  when: kafka_version is version('2.0', '>')
```

Comment:
now you'll need to add another task for the old log properties:

```yaml
- name: Copies kafka log4j.properties (0.8.2.2)
  copy:
    src: "log4j_0.8.2.2.properties"
    dest: /opt/kafka/config/log4j.properties
    owner: root
    group: root
    mode: 644
  when: kafka_version == "0.8.2.2"
  tags: log4j
```

Reply:
added this too